poc/tests/test_field.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from vdaf_poc.field import NttField
+
+from field32 import Field32
+from field40 import Field40
+
+
```

Comment:
Make a note here that this is copied from github.com/cfrg/draft-irtf-cfrg-vdaf and eventually we should refactor so that we use code from there.